packages/docs-ui/README.md

```diff
@@ -9,15 +9,20 @@ Components for building documentation sites with a consistent user experience to
 npm install @braid-design-system/docs-ui
 ```
 
-- [LinkableHeading](#linkableheading)
-- [MenuButton](#menubutton)
+- [docs-ui](#docs-ui)
+  - [Usage](#usage)
+    - [`LinkableHeading`](#linkableheading)
+      - [Example](#example)
+      - [Props](#props)
+    - [`MenuButton`](#menubutton)
+      - [Example](#example-1)
+      - [Props](#props-1)
```

Comment:
Looks like I accidentally added a table of contents generated by [this extension](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one). Do we want to keep it?